docs/install/install-include.rst

```diff
@@ -57,90 +52,54 @@ Select your preferences and run the install command.
 
      .. container:: stable
 
-        .. container:: native
+        .. container:: cpu-only
 
            .. code-block:: bash
 
-              pip install --upgrade mxnet gluonnlp
+              # Install MXNet
+              python3 -m pip install -U --pre "mxnet>=2.0.0" -f https://dist.mxnet.io/python
 
-        .. container:: cuda
-
-           .. code-block:: bash
-
-              # Here we assume CUDA 10.0 is installed. You can change the number
-              # according to your own CUDA version.
-              pip install --upgrade mxnet-cu100 gluonnlp
-
-        .. container:: mkl-dnn
-
-           .. code-block:: bash
-
-              pip install --upgrade mxnet-mkl gluonnlp
-
-        .. container:: cuda-mkl-dnn
-
-           .. code-block:: bash
-
-              # Here we assume CUDA 10.0 is installed. You can change the number
-              # according to your own CUDA version.
-              pip install --upgrade mxnet-cu100mkl gluonnlp
-
-     .. container:: nightly
-
-        .. container:: native
-
-           .. code-block:: bash
-
-              pip install --pre --upgrade mxnet https://github.com/dmlc/gluon-nlp/tarball/master
+              # Install GluonNLP
+              git clone https://github.com/dmlc/gluon-nlp.git
+              cd gluon-nlp
+              python3 -m pip install -U -e ."[extras]"
 
         .. container:: cuda
 
            .. code-block:: bash
 
-              pip install --pre --upgrade mxnet-cu100 https://github.com/dmlc/gluon-nlp/tarball/master
-
-        .. container:: mkl-dnn
-
-           .. code-block:: bash
-
-              pip install --pre --upgrade mxnet-mkl https://github.com/dmlc/gluon-nlp/tarball/master
-
-        .. container:: cuda-mkl-dnn
-
-           .. code-block:: bash
+              # Here we assume CUDA 10.2 is installed. You can change the number
+              # according to your own CUDA version, e.g., cu101, cu110
+              python3 -m pip install -U --pre "mxnet-cu102>=2.0.0" -f https://dist.mxnet.io/python
```

Comment:
As per ASF policy, we may need a disclaimer to point out that this is a unreleased nightly build and should only be used for the purpose of development by the community and isn't intended for general public. We can replace it with a link to the official alpha / beta release once it passed vote